yt/frontends/ramses/particle_handlers.py

```diff
@@ -282,3 +289,36 @@ def read_header(self):
         self.field_offsets = field_offsets
         self.field_types = _pfields
         fd.close()
+
+
+class SinkParticleFileHandlerCsv(ParticleFileHandler):
+    """Handle sink files from a csv file, the format from the sink particle in ramses"""
+
+    ptype = "sink_csv"
+    fname = "sink_{iout:05d}.csv"
+    file_descriptor = None
+    config_field = "ramses-sink-particles"
+    reader = _ramses_particle_csv_file_handler
+    attrs = (("nsink", 1, "i"), ("nindsink", 1, "i"))
+    second_read = False
```

Comment:
this doesn't seem to be used ?
```suggestion
```

Reply:
yes, I'm not using it as we can read directly the all file